docs/installation/arch.rst

```diff
@@ -1,20 +1,19 @@
 .. _arch-install:
 
-****************************
-Arch Linux: Install from AUR
-****************************
+**************************************
+Arch Linux: Install from ``[community]``
+**************************************
```

Comment:
I've noticed a couple of Arch users talking about "[community]", but it seems just as usual to just write "community" or "community repository". Can we use one of those variants and avoid this quite special form, at least in the headers?

Also, the length of the `*` lines doesn't match the text length.